k8s/agones/Makefile

```diff
@@ -24,8 +24,7 @@ APP_PARAMETERS ?= { \
 }
 
 
-APP_TEST_PARAMETERS ?= { \
-}
+APP_TEST_PARAMETERS ?= { }
```

Comment:
we should just remove this altogether to prevent someone from innocently adding values here instead of the test schema.